apollo-router/src/plugins/telemetry/filtering_layer.rs

```diff
@@ -0,0 +1,141 @@
+use std::collections::HashMap;
+use std::collections::HashSet;
+
+use tracing::field;
+use tracing::Level;
+use tracing::Subscriber;
+use tracing_core::Field;
+use tracing_subscriber::Layer;
+
+// Specific attributes for logging
+pub(crate) const SPECIFIC_ATTRIBUTES: [&str; 4] = [
```

Comment:
Seconding @Geal comment about this name "SPECIFIC" seems odd. If you are trying to avoid "DEFAULT", then perhaps consider "USUAL" or "REQUIRED" (if they are required).

Reply:
Go for `DEFAULT` ! Thanks :)